utils/metrics.py

```diff
@@ -64,14 +64,16 @@ def report_classification(pred, label, batch_size, metrics_dict, ignore_index=-1
             metrics_dict['recall_' + key].update(class_score[key]['recall'], batch_size)
             metrics_dict['fscore_' + key].update(class_score[key]['f1-score'], batch_size)
 
+
     metrics_dict['precision'].update(class_report['weighted avg']['precision'], batch_size)
     metrics_dict['recall'].update(class_report['weighted avg']['recall'], batch_size)
     metrics_dict['fscore'].update(class_report['weighted avg']['f1-score'], batch_size)
+    metrics_dict = iou(pred, label, batch_size, metrics_dict)
```

Comment:
Just maybe we need to talk about the implementation and how it's call, make it similar to the lines before.